src/test/tests/databases/blueprint.py

```diff
@@ -76,6 +77,16 @@
 venn_s_by_m_yaml_root  =  data_path(pjoin(bp_venn_test_dir,
                                "venn_small_example_sparse_by_material_yaml.root"))
 
+bp_part_map_root = data_path(pjoin(bp_part_map_test_dir,
+                             "tout_custom_part_map_index_hdf5.root"))
+
+bp_part_map_single_file_root = data_path(pjoin(bp_part_map_test_dir,
+                                         "tout_single_file_part_map_index_hdf5.root"))
+
+bp_sprial_part_map_examples = []
+for i in range(-1,9):
+    bp_sprial_part_map_examples.append(data_path(pjoin(bp_part_map_test_dir,
+                                       "tout_relay_sprial_mesh_save_nfiles_{0}.cycle_000000.root".format(i))))
```

Comment:
Could change to spiral but it doesn't matter

Reply:
lol, i guess I guess its misspelled in conduit ...

Reply:
I fixed this, thanks for pointing this out.